integration/cli-tests/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/api/cliTestIterateMediatorApi.xml

```diff
@@ -0,0 +1,64 @@
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  ~
+  -->
+
+<api xmlns="http://ws.apache.org/ns/synapse" name="cliTestIterateMediatorApi" context="/iteratesample8">
+        <resource methods="POST" uri-template="/iteratejson8">
+            <inSequence>
+                <log level="full"/>
+                <iterate id="jsonIterator8" preservePayload="true" attachPath="json-eval($.students.studentlist)" expression="json-eval($.students.studentlist)" sequential="true">
```

Comment:
Let's not create complicated artifacts. Just having a respond mediator would be enough IMO. WDYT?

Reply:
+1